js/passwordManager/keychain.js

```diff
@@ -48,6 +48,53 @@ class Keychain {
     ipcRenderer.invoke('credentialStoreDeletePassword', { domain, username })
   }
 
+  async importCredentials () {
```

Comment:
I think it would be better if this took the CSV file contents as an argument, and the open dialog part was moved into passwordViewer. It's possible the CSV parsing would be better moved into there as well, but I'm not sure about that, so I'll leave that up to you.

Reply:
I was reading this comment and was like, "huh but this is already in the viewer right?" apparently not, thanks for catching :)